terraform/install.sh

```diff
@@ -39,24 +43,59 @@ applyTerraform() {
 
 getExternalIp() {
   external_ip=""; 
-  while [ -z $external_ip ]; 
-  do
+  while [ -z $external_ip ]; do
      log "Waiting for Hipster Shop endpoint..."; 
      external_ip=$(kubectl get svc frontend-external --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}"); 
      [ -z "$external_ip" ] && sleep 10; 
   done; 
-
-  log "Verifying that Hipster Shop Frontend is accessible"
-  if [[ $(curl -sL -w "%{http code}\\n" "http://$external_ip" -o /dev/null) -eq 200 ]]
-  then
-    log "Stackdriver Sandbox cluster provisioning has completed successfully! Access it at http://$external_ip"
-  fi
+  log "Hipster Shop Frontend is available at $external_ip"
 }
 
 # Install Load Generator service and start generating synthetic traffic to Sandbox
 loadGen() {
   log "Running load generator"
   ../loadgenerator-tool startup --zone us-west2-a $external_ip
+  # find the IP of the load generator web interface
+  TRIES=0
+  while [[ -z "${loadgen_ip}" && "${TRIES}" -lt 20  ]]; do
```

Comment:
Is it taking so long that you decided on 20 retries?
Can we reduce it to, say, 10?

Reply:
We could. I thought since it's sleeping for 1 second each loop, 20 seconds would be a good upper limit, but I can change it to 10 if you prefer

Reply:
Up to you.